tool/travis.sh

```diff
@@ -142,30 +142,27 @@ elif [ "$BOT" = "flutter_sdk_tests" ]; then
 
 elif [ "$BOT" = "packages" ]; then
 
-    pub global activate tuneup
-
-    # Analyze packages/devtools_server.
     popd
-    pushd packages/devtools_server
-    echo `pwd`
 
-    pub get
-    pub global run tuneup check
+    pub global activate tuneup
 
-    # Analyze third_party/packages/split.
-    popd
-    pushd third_party/packages/split
-    echo `pwd`
+    # Analyze packages/
+    (cd packages/devtools_app; pub get)
+    (cd packages/devtools_server; pub get)
+    (cd packages/devtools_testing; pub get)
+    (cd packages/html_shim; pub get)
+    (cd packages; pub global run tuneup check)
```

Comment:
does this call run `pub global run tuneup check` for each of the above directories?

Reply:
Yup, it'll recurse down into subdirectories and analyze them.